app/views/layouts/_user_avatar.erb

```diff
@@ -11,5 +11,5 @@
         <%= classes %>
         "
 >
-  <%= user.name.initials&.upcase %>
+  <%= only_user_initials(user.name.initials).upcase %>
```

Comment:
The method name is a bit confusing. I find myself thinking, “but isn’t name.initialis” only the user initials? I don’t expect that to return something other than initials.

A better name might be: “only_two_initials”

Reply:
only_two_initials makes great sense

Reply:
only_two_initials was an even better name but I went with at_most_two_initials to cover the case for one initial